test/trimCharsStart.js

```diff
@@ -0,0 +1,33 @@
+import { assert } from 'chai';
+
+import * as RA from '../src';
+
+describe('trimCharsStart', function() {
+  it('should remove all occurences of a trim character', function() {
```

Comment:
Pls don't use currying in tests. We'll have a separate test that is testing currying itself.
```js
assert.strictEqual(RA.trimCharsStart('-', '---abc'), 'abc');